### ISSUE TITLE
poly.order connects all vowels, not just those specified

### ISSUE THREAD

**Issue by valerief** [drammock/phonR#36]
The phonR documentation for `poly.order` says, "If there are values of vowel not included in poly.order, they will not be connected to the polygon line," but in the below plot, the line connects the vowels I specify, and then continues, connecting the rest in alphabetical order. 

My input (output attached below): 
```
plotVowels(f1=goodmidpts$nry2_vowel_f1_pt50, 
           f2=goodmidpts$nry2_vowel_f2_pt50, 
           vowel=goodmidpts$vowel_label, group=goodmidpts$sex, 
           plot.tokens=F, plot.means=T, 
           pch.means=goodmidpts$vowel_label, 
           poly.line=T, poly.fill=F, 
           poly.order=c("IY", "EY", "AE", "AA", "AO", "OW", "UW"),
           var.col.by=goodmidpts$sex, var.style.by=NULL, 
           legend.kwd="bottomleft", col=c("red3", "blue3"),
           ylim=c(-0.2,-1.0), xlim=c(1.0,0.2), pretty=T)
```
![wonky_polylines](https://cloud.githubusercontent.com/assets/6989089/7208837/60f1fd32-e4f8-11e4-9028-6a860c552ab0.jpg)

**Comment by drammock** [comment]
This is now fixed in version 1.0-3, which is now on GitHub. This one is serious enough that I'll probably submit an updated version to CRAN, though it won't be for a few days yet.

**Comment by valerief** [comment]
Looks great, thanks!
